README.md

```diff
@@ -1,48 +1,34 @@
 # Cardano-Swaps
+A [p2p-DeFi protocol](https://github.com/zhekson1/CSL-DeFi-Protocols) for swapping fungible tokens on the Cardano Settlement Layer
 
-:warning: Knowledge of basic Haskell syntax and cardano-cli usage is assumed.
+> **Note**
+> Knowledge of basic Haskell syntax and cardano-cli usage is recommended. For a list of everything that has changed from the previous version, see the [Changelog](CHANGELOG.md).
```

Comment:
I think the comments about the performance changes should be here too that way people don't need to re-read the entire thing just to get the changes. The changelog does not cover the benchmarks.

Reply:
don't think it should be at the very beginning of the doc. moved it a bit further down